apps/FreeCAD/install-64

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+
+version=1.0.2
+
+wget -O /tmp/FreeCAD.AppImage "https://github.com/FreeCAD/FreeCAD/releases/download/${version}/FreeCAD_${version}-conda-Linux-aarch64-py311.AppImage" || exit 1
+
+sudo mv /tmp/FreeCAD.AppImage /opt/FreeCAD.AppImage || error "Failed to move AppImage to /opt"
+
+status -n "Installing AppImage to /opt, making it executable, adding symlink... "
+sudo chmod +x /opt/FreeCAD.AppImage || error "Failed to mark as executable!"
+# Make terminal command
+sudo rm -f /usr/local/bin/FreeCAD
+sudo ln -s /opt/FreeCAD.AppImage /usr/local/bin/FreeCAD || error "Failed to make /usr/local/bin/FreeCAD terminal symlink!"
+status_green Done
+
+
+### DOWNLOAD AND INSTALL OTHER ASSETS
+
+# Download .desktop file.
+status "Downloading .desktop file..."
+wget -O /tmp/org.freecad.FreeCAD.desktop "https://github.com/FreeCAD/FreeCAD/raw/refs/tags/$version/src/XDGData/org.freecad.FreeCAD.desktop" || exit 1
+sudo mv /tmp/org.freecad.FreeCAD.desktop /usr/share/applications/org.freecad.FreeCAD.desktop || error 'Unable to move .desktop file from tmp'
+
+# Download and install icon
+status "Downloading icon..."
+wget -O /tmp/org.freecad.FreeCAD.svg "https://github.com/FreeCAD/FreeCAD/raw/refs/tags/$version/src/Gui/Icons/freecad.svg" || exit 1
+sudo mv /tmp/org.freecad.FreeCAD.svg /usr/share/pixmaps/org.freecad.FreeCAD.svg || error 'Failed to install the icon.'
+
+# Download mimetype xml file
+status "Downloading mimetype file..."
+wget -O /tmp/freecad.xml "https://github.com/FreeCAD/FreeCAD/raw/refs/tags/$version/src/XDGData/org.freecad.FreeCAD.xml" || exit 1
```

Comment:
@Botspot @ooovlad no need to download any of these, they are all available in the appimage itself. if you want to install them to the system, you can simply extract and move them to the correct place

```
./FreeCAD_weekly-2025.09.17-Linux-x86_64-py311.AppImage --appimage-extract usr/share/mime/packages/org.freecad.FreeCAD.xml
./FreeCAD_weekly-2025.09.17-Linux-x86_64-py311.AppImage --appimage-extract org.freecad.FreeCAD.svg
./FreeCAD_weekly-2025.09.17-Linux-x86_64-py311.AppImage --appimage-extract org.freecad.FreeCAD.desktop
```


```bash
$ tree squashfs-root
squashfs-root
├── org.freecad.FreeCAD.desktop
├── org.freecad.FreeCAD.svg
└── usr
    └── share
        └── mime
            └── packages
                └── org.freecad.FreeCAD.xml

5 directories, 3 files
```